jaeger_client/constants.py

```diff
@@ -47,6 +47,9 @@
 # Tracer-scoped tag that contains the hostname
 JAEGER_HOSTNAME_TAG_KEY = 'jaeger.hostname'
```

Comment:
Does this need to have jaeger. prefix?

Reply:
Go, Cpp, and Java use `hostname` and this and node use `jaeger.hostname`. I can go with `hostname`

Reply:
let's change

Reply:
`jaeger.hostname` made sense with Zipkin model when it was transferred to span tags, but in the `Process` the prefix is not necessary.